docs/reference/commandline/service_create.md

```diff
@@ -546,6 +546,7 @@ x3ti0erg11rjpg64m75kej2mz-hosttempl
 ## Related information
 
 * [service inspect](service_inspect.md)
+* [service logs](service_logs.md)
 * [service ls](service_ls.md)
 * [service rm](service_rm.md)
 * [service scale](service_scale.md)
```

Comment:
@allencloud yeah, it's not a "pretty" solution, but in this case it worked :smile: